proxy/secret.go

```diff
@@ -0,0 +1,66 @@
+package proxy
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/openfaas/faas-cli/schema"
+)
+
+//GetSecretList get secrets list
+func GetSecretList(gateway string, tlsInsecure bool) ([]schema.Secret, error) {
+	var results []schema.Secret
+
+	if tlsInsecure {
+		if !strings.HasPrefix(gateway, "https") {
+			fmt.Println("WARNING! Communication is not secure, please consider using HTTPS. Letsencrypt.org offers free SSL/TLS certificates.")
+		}
+	}
+
+	gateway = strings.TrimRight(gateway, "/")
+	client := MakeHTTPClient(&defaultCommandTimeout, tlsInsecure)
+
+	getRequest, err := http.NewRequest(http.MethodGet, gateway+"/system/secrets", nil)
+	SetAuth(getRequest, gateway)
+
+	if err != nil {
+		return nil, fmt.Errorf("cannot connect to OpenFaaS on URL: %s", gateway)
+	}
+
+	res, err := client.Do(getRequest)
+	if err != nil {
+		return nil, fmt.Errorf("cannot connect to OpenFaaS on URL: %s", gateway)
+	}
+
+	if res.Body != nil {
+		defer res.Body.Close()
+	}
+
+	switch res.StatusCode {
+	case http.StatusOK:
```

Comment:
Please add http.StatusAccepted too.

Reply:
will add it. just a doubt will`GET` method return status`http.StatusAccepted`?